<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <widget class="QWidget" name="ugqids">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="layout_qrcode">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请使用bilibili客户端扫描二维码登录</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
